CoastalResilienceNetwork/GeositeFramework
# Remove "zoom_to" plugin and add ESRI search control

Replaces the `zoom_to` plugin with the ESRI search control.

![image](https://cloud.githubusercontent.com/assets/1042475/19656305/6fd7a724-99ed-11e6-8907-3bc71ac3e30e.png)

**Testing**
- Visit the app and try out the search control in the header.
- Verify all references to the `zoom_to` plugin have been removed.

**Notes**
There are some display issues on small screens. I'm going to update https://github.com/CoastalResilienceNetwork/GeositeFramework/issues/690.

Connects to #701
